work-list-page.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
```

Comment:
- [ ] REQUIRED: For your next review, kindly please include only ONE HTML file which includes all your work for all the milestones up until now, so the next reviewers have an easier time working on your review request.  It should include all the sections including the "about myself".   The "about myself" should meet all the requirements (mind your font styles, your background-color, your background decorations, etc).

You may need to adjust your CSS file too.

That way, we will be able to approve your project faster 🥇